documentation/src/main/asciidoc/topics/upgrading.adoc

```diff
@@ -1,5 +1,10 @@
 = Upgrading from 12.0 to 12.1
 
+== Search
+
+The optional ```IndexedQueryMode``` parameter has been removed from queries in library mode, CLI, Hot Rod and REST.
```

Comment:
shouldn't `IndexedQueryMode` has been deprecated first?

Reply:
Ideally yes, but since 11 or 12 it's being ignored internally